src/drawer/drawer.tsx

```diff
@@ -61,34 +79,45 @@ export const Drawer = ({
             enableOverlayClick
             onOverlayClick={onOverlayClick}
         >
-            <Container
-                $show={show}
-                data-testid="drawer"
-                onClick={handleClick}
-                role="dialog"
-                aria-labelledby={id}
-                onTransitionEnd={handleDialogVisibility}
-                {...otherProps}
+            <FloatingFocusManager
```

Comment:
FloatingFocusManager should only be rendered when the drawer is active per the fix here https://github.com/LifeSG/react-design-system/pull/788/files

if conditional rendering breaks the animation, take a look at `useTransitionStatus()` to only unmount when the animation completes https://floating-ui.com/docs/usetransition#usetransitionstatus